src/app/cases/case.resolver.ts

```diff
@@ -59,16 +62,29 @@ export class CaseResolver implements Resolve<CaseView> {
   }
 
   private getAndCacheCaseView(jid, ctid, cid): Observable<CaseView> {
+    if (cid.startsWith(CaseResolver.DRAFT)) {
```

Comment:
couldn't we have a method on CaseDetails isDraft(cid) ?

Reply:
addressed in the next [PR](https://github.com/hmcts/ccd-case-management-web/pull/183)